packages/react/src/components/Flex/FlexItem.tsx

```diff
@@ -0,0 +1,109 @@
+import * as React from 'react'
+import * as PropTypes from 'prop-types'
+import cx from 'classnames'
+import { UIComponent, commonPropTypes } from '../../lib'
+import { mergeStyles } from '../../lib/mergeThemes'
+import { Extendable } from '../../types'
+
+export type FlexItemSize = 'size.half' | 'size.quarter' | 'size.small' | 'size.medium'
```

Comment:
How did we ended up with this values for size? Again confused, why we have values like small and medium, but not big? What these values mean?

Reply:
for now just those that were used on the examples were introduced. There is no spec on the values now, it was rather about approach to values introduced for now. However, just to maintain consistency with the rest of the API, lets introduce the same range of size values (`smallest` -> `largest`)